Engine/src/Engine/ShaderGenerator.cpp

```diff
@@ -0,0 +1,64 @@
+#include "ShaderGenerator.h"
+#include "Log.h"
+#include "glad/glad.h"
+
+namespace Engine
+{
+	ShaderGenerator::ShaderGenerator(const char* vertexSource, const char* fragmentSource)
```

Comment:
Is this the generic class for making shaders?  Maybe a couple comments on its purpose and use could help